src/components/AdminInvite.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import {
+  FormControl,
+  Image,
+  FormLabel,
+  InputGroup,
+  Input,
+  InputRightAddon,
+  Button,
+  Flex,
+  Link,
+  Center,
+} from '@chakra-ui/react';
+import { sendInviteLink } from '../utils/auth_utils';
```

Comment:
🚫 **[eslint]** <[import/named](https://github.com/import-js/eslint-plugin-import/blob/v2.25.4/docs/rules/named.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>sendInviteLink not found in '../utils/auth_utils'